pokemon.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<title>Pokemons</title>
+		<link rel="stylesheet" href="Normalization.css" />
+		<link rel="stylesheet" href="pokemon.css" />
+		<link rel="preconnect" href="https://fonts.googleapis.com" />
+		<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+		<link
+			href="https://fonts.googleapis.com/css2?family=Tilt+Prism&display=swap"
+			rel="stylesheet" />
+		<script defer src="pokemon.js"></script>
```

Comment:
mimo że jest defer to i tak bym przerzucił to na sam dół dokumentu, wtedy mamy pewność że wszystko się wczyta